vendor/pchart/pChart.class.php

```diff
@@ -208,7 +208,7 @@ public function __construct($XSize,$YSize)
        }
 
        $C_White =$this->AllocateColor($this->Picture,255,255,255);
-       imagefilledrectangle($this->Picture,0,0,$XSize,$YSize,$C_White);
+       imagefilledrectangle($this->Picture,0,0,(int) round($XSize),(int) round($YSize),$C_White);
```

Comment:
if $XSize is not an int or a float : it broke (null maybe)

`rount(intval($XSize))` seems the best